getting-started-gradle/src/main/java/quickstart/GreetingResource.java

```diff
@@ -0,0 +1,28 @@
+package quickstart;
```

Comment:
could we go with packages similar to the other projects? `org.acme.gradle` would be best. Same for the tests.

Reply:
Ok :)